products/notebooks/api.yaml

```diff
@@ -247,6 +249,7 @@ objects:
           - STOPPING
           - STOPPED
           - DELETED
+          - INITIALIZING
```

Comment:
Honestly since this is output-only, I'd recommend not documenting the allowed values and just making it a string instead of an enum. That makes it easier to adjust when the API adds more values, and we don't have any reason to validate them since users can't set them.